### ISSUE TITLE
Can't edit Catalogs as Catalog Manager

### ISSUE THREAD

**Issue by tpinkowski** [ampache/ampache#1184]
# Environment
- Ampache version: 3.8.2
- Web server + version: Apache 2.2
- Server operating system: Linux(CentOS 6.5)
- Client operating system: Windows
- Client type:
  - [ x] Web
  - Client application + version: Firefox 45.0.1
# Description

Hi,
I have created a user and assigned the role "Catalog Manager" to him.
But I have no possibility on the left navigation or a menu to edit Catalogs on the Frontend.
Am I missing something or do I need to do some additional configurations to get it work?

**Comment by tpinky** [comment]
Would yo accept a pull request with following modifications:

- in sidebar_home.inc.php
```
<?php
    if (Access::check('interface', '75')) {
        ?>
        
        <li><h4 class="header"><span class="sidebar-header-title" title="<?php echo T_('Catalogs'); ?>"><?php echo T_('Catalogs'); ?></span><img src="<?php echo AmpConfig::get('web_path') . AmpConfig::get('theme_path'); ?>/images/icons/icon_all.png" class="header-img <?php echo isset($_COOKIE['sb_catalogs']) ? $_COOKIE['sb_catalogs'] : 'expanded'; ?>" id="catalogs" alt="<?php echo T_('Expand/Collapse'); ?>" title="<?php echo T_('Expand/Collapse'); ?>" /></h4>
            <ul class="sb3" id="sb_admin_catalogs">
                <li id="sb_admin_catalogs_Add"><a href="<?php echo $web_path; ?>/admin/catalog.php?action=show_add_catalog"><?php echo T_('Add a Catalog'); ?></a></li>
                <li id="sb_admin_catalogs_Show"><a href="<?php echo $web_path; ?>/admin/catalog.php?action=show_catalogs"><?php echo T_('Show Catalogs'); ?></a></li>
            </ul>
        </li>
        <?php
    }
    ?>
```
- in catalog.php
```
if (!Access::check('interface','100') && !Access::check('interface','75')) {
    UI::access_denied();
    exit;
}
```

- in sse.server.php
```
if (!Access::check('interface','100') && !Access::check('interface','75')) {
    UI::access_denied();
    exit;
}
```

**Comment by Phyks** [comment]
Does this change solve the issue? Could you submit a PR so that we could discuss the fix more easily?

**Comment by lachlan-00** [comment]
I've tested a fix and put a commit into #1957.
